apps/web/lib/hooks/useRouterQuery.ts

```diff
@@ -5,7 +5,19 @@ export default function useRouterQuery<T extends string>(name: T) {
   const existingQueryParams = router.asPath.split("?")[1];
 
   const urlParams = new URLSearchParams(existingQueryParams);
-  const query = Object.fromEntries(urlParams);
```

Comment:
fromEntries returns Record<string,string> value and thus it ends up overriding existing keys.